apps/editing-toolkit/editing-toolkit-plugin/help-center/src/help-center.scss

```
@@ -20,3 +21,49 @@
 		}
 	}
 }
+
+/**
+ * WP-ADMIN Masterbar Styling
+ */
+.wp-toolbar {
+	#wpadminbar #wp-admin-bar-help .ab-item {
+		position: relative;
+		padding: 0;
+		width: 36px;
+
+		&:hover {
+			cursor: pointer;
+		}
+	}
+
+	svg#help-center__icon {
+		position: absolute;
+		top: 0;
+		left: 0;
+		right: 0;
+		bottom: 0;
+		margin: auto;
```

We should use padding here instead of margin to make the entire thing clickable. If you click on the edges nothing happens now. 

```suggestion
		padding: 4px 6px;
```

---

Got it!